Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_state_cache.mm

```diff
@@ -699,19 +684,21 @@ MTLColorWriteMask AdjustColorWriteMaskForSharedExponent(MTLColorWriteMask mask)
 void RenderPassAttachmentDesc::reset()
 {
     texture.reset();
-    implicitMSTexture.reset();
-    level              = mtl::kZeroNativeMipLevel;
-    sliceOrDepth       = 0;
-    blendable          = false;
-    loadAction         = MTLLoadActionLoad;
-    storeAction        = MTLStoreActionStore;
-    storeActionOptions = MTLStoreActionOptionNone;
+    resolveTexture.reset();
+    level               = mtl::kZeroNativeMipLevel;
+    sliceOrDepth        = 0;
+    resolveLevel        = mtl::kZeroNativeMipLevel;
+    resolveSliceOrDepth = 0;
```

Comment:
Now the state for each target is explicitly for the one it's intended to be used with.